apidoc/Titanium/UI/Window.yml

```
@@ -923,7 +923,7 @@ properties:
         Read more about the safe-area layout-guide in the [Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/overview/iphone-x/).
     platforms: [android, iphone, ipad]
     type: Boolean
-    default: {android: false, ios: true}
+    default: false on Android, true on iOS.
```

To get it to show the raw js value literal as monospace/code, wrap in backticks
```suggestion
    default: `false` on Android, `true` on iOS.
```